frontend/src/pages/RecipientRecord/pages/Profile.js

```diff
@@ -9,66 +9,70 @@ import './Profile.css';
 import FeatureFlag from '../../../components/FeatureFlag';
 import ClassReview from './components/ClassReview';
 import MonitoringReview from './components/MonitoringReview';
-import { GrantDataProvider } from './GrantDataContext';
+import { useGrantData } from './GrantDataContext';
+import { ClassGuidanceProvider } from './ClassGuidanceContext';
 
 export default function Profile({
   recipientSummary,
   regionId,
   recipientId,
 }) {
   const activeGrants = recipientSummary.grants.filter((grant) => grant.status === 'Active');
+  const { hasMonitoringData, hasClassData } = useGrantData();
 
   return (
     <>
       <Helmet>
         <title>Profile</title>
       </Helmet>
-      <GrantDataProvider>
+      <ClassGuidanceProvider>
         <div className="maxw-widescreen">
           <Grid row gap={4}>
             <Grid desktop={{ col: 3 }} tabletLg={{ col: 12 }}>
-              <RecipientSummary summary={recipientSummary} regionId={regionId} />
+              <RecipientSummary summary={recipientSummary} />
             </Grid>
             <Grid desktop={{ col: 9 }} tabletLg={{ col: 12 }}>
-              <RecipientLeadership recipientId={recipientId} regionId={regionId} />
```

Comment:
This seems to require regionId to work

Reply:
👍 50d72b6869213aa6bd1a7fd333c08650f5069daa